cypress/cypress/support/commands.ts

```diff
@@ -43,12 +43,12 @@ addMatchImageSnapshotCommand({
 });
 
 // recursively gets an element, returning only after it's determined to be attached to the DOM for good
-Cypress.Commands.add("getSettled", (selector, opts = {}) => {
+Cypress.Commands.add("getSettled", (selector: any, opts = {}) => {
```

Comment:
Was this to solve the "getSettled" issue of not being a known function?
If so, it should be fixed with this approach (which allows you to type the parameters for the command, instead of just using any): https://docs.cypress.io/guides/tooling/typescript-support#Types-for-Custom-Commands